docker/SERVER/docker-compose.yaml

```diff
@@ -0,0 +1,63 @@
+version: '3.8'
+services:
+  nginx-proxy:
+    image: nginx:1.21.6
+    container_name: nginx-proxy
+    ports:
+      - "80:80"
+      - "443:443"
+    volumes:
+      - conf:/etc/nginx/conf.d
+      - vhost:/etc/nginx/vhost.d
+      - html:/usr/share/nginx/html
+      - certs:/etc/nginx/certs:ro
+      - ./custom_settings.conf:/etc/nginx/conf.d/custom_settings.conf:ro
+    labels:
+      - "com.github.jrcs.letsencrypt_nginx_proxy_companion.nginx_proxy"
+    restart: always
+    networks: 
+      - commonnetwork
+      
+  docker-gen:
+    image: jwilder/docker-gen:0.8.4
```

Comment:
What is the reason for using nginx directly and docker-gen, as oppose to continue to rely on nginx-proxy?

Reply:
It's a well tested setup I use on my servers.

Using docker-gen, you're able to separate and maintain any version of nginx at any time.  It's highly customizable using the template file.

If you don't like this approach, I can revert and get the network to be created automatically on nginx-proxy.

Please let me know what you prefer and I'll move forward with that.

Reply:
Would prefer the nginx-poxy image, it's kept up-to-date and less custom code needed for us? https://github.com/nginx-proxy/nginx-proxy

Reply:
Ok, no problem.